### PR TITLE
Create cfp-close.md

### PR DESCRIPTION
**Pull request by Cheukting**

### CODE REVIEW

File: content/news/cfp-close.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: "Thank you ❤️ PyLadiesCon 2024 received many proposals!"
+meta_title: "Call for Proposal of PyLadiesCon 2024 was closed and we have received almost 200 proposals!"
+description: "Call for Proposal of PyLadiesCon 2024 was closed on 15th September and we have received almost 200 proposaled! Thank you for all that has been submitted, in our minds you are super awesome and we would love to see your talk. However, the reviews will have a busy weeks ahead to review all the proposals and we will have tough decisions to make regarding which talks to accept. We aim to publish a tentative schedule by 8th October.
```
Comment (Mariatta) [inline content/news/cfp-close.md:4]:
```suggestion
description: "Call for Proposal of PyLadiesCon 2024 was closed on 15th September and we have received almost 200 proposals! Thank you for all that has been submitted, in our minds you are super awesome and we would love to see your talk. However, the reviews will have a busy weeks ahead to review all the proposals and we will have tough decisions to make regarding which talks to accept. We aim to publish a tentative schedule by 8th October.
```

File: content/news/cfp-close.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+"
+date: 2024-09-17T05:00:00Z
+categories: ["Blog Post",]
+author: "Cheuk"
+tags: ["python", "community", "conference", "public speaking", "pyladies"]
+draft: false
+layout: news
+---
+
+Call for Proposal of PyLadiesCon 2024 was closed on 15th September and we have received almost 200 proposaled! Thank you for all that has been submitted, in our minds you are super awesome and we would love to see your talk. However, the reviews will have a busy weeks ahead to review all the proposals and we will have tough decisions to make regarding which talks to accept. We aim to publish a tentative schedule by 8th October.
```
Comment (Mariatta) [inline content/news/cfp-close.md:14]:
```suggestion
Call for Proposal of PyLadiesCon 2024 was closed on 15th September and we have received almost 200 proposals! Thank you for all that has been submitted, in our minds you are super awesome and we would love to see your talk. However, the reviews will have a busy weeks ahead to review all the proposals and we will have tough decisions to make regarding which talks to accept. We aim to publish a tentative schedule by 8th October.
```

File: content/news/cfp-close.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+- PyLadies Berlin
+- PyLadies Taiwan
+- PyLadies Maputo
+- PyLadies Colombia
+- PyLadies Santo Domingo
+- PyLadies São Carlos
+
+### Sepaker mentorship ❤️
+
+We have also, for the first time, provide mentorships to those who are new in speaking at conferences and submitting proposals. The first round of submittion has been closed on 15th September. We received 20 application for finding a mentor and we have on boarded 7 mentors. While we cannot assign everyone a mentor, we have match some mentors to our mentee based on the language that they speak, their avaliable timezone and their expertees. Thank you for the following mentors who help bringing new speakers to the communiy:
```
Comment (Mariatta) [inline content/news/cfp-close.md:39]:
```suggestion
We have also, for the first time, provided mentorships to those who are new in speaking at conferences and submitting proposals. The first round of submission has been closed on 15th September. We received 20 applications for finding a mentor, and we have onboarded 7 mentors. While we cannot assign everyone a mentor, we have matched some mentors to our mentee based on the language that they speak, their available timezone, and their expertise. Thank you for the following mentors who help bringing new speakers to the community:
```

File: content/news/cfp-close.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+We will try to run another round of speaker mentorship when the speakers have been selected and help new speakers get a mentor to help prepare their talks.
+
+Other than selecting and scheduling talks, we will be planning other activities during the conference. Last year, we had panels, discussion sessions, and much more. If you have any great ideas, please feel free to email us at: [pyladiescon@pyladies.com](pyladiescon@pyladies.com)
+
+### We need your support
+
+To make PyLadiesCon a success and to support PyLadies, please consider talking to your employer about [sponsoring PyLaidescon 2024](https://conference.pyladies.com/sponsors/). PyLadiesCon is an annual gathering of all PyLadies globally, this is a great opportunity to give back to the community that is dedicated to supporting the underrepresented groups in the tech community.
+
+Last, we will recap the timeline leading to PyLadiesCon 2024, please look forward to the conference:
```
Comment (Mariatta) [inline content/news/cfp-close.md:57]:
```suggestion
Here's our timeline leading to PyLadiesCon 2024.
Please look forward to the conference.
```

File: content/news/cfp-close.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+Despite the effort to promote more diversity languages and the support of many PyLadies Chapters, there are still not a lot of submissions besides English and Español. In the future, more strategic planning regarding which language we would like to invite folks to submit would be needed to plan ahead. Nevertheless, a big thanks to the following PyLadies Chapters for helping us during the Call for Proposal periods:
+
+- PyLadies Tokyo
+- PyLadies Berlin
+- PyLadies Taiwan
+- PyLadies Maputo
+- PyLadies Colombia
+- PyLadies Santo Domingo
+- PyLadies São Carlos
+
```
Comment (Mariatta) [inline content/news/cfp-close.md:36]: Can we also mention thanks to people who helped by sharing and boosting our CFP?
```suggestion

And thank you to everyone who supported our CfP by sharing our news and posts on social media. It is with your help that we're able to break this record.
```

**Review by Mariatta — COMMENTED** [review]
Looks great! Thank you.

I left some comments but mostly just typos.

**Comment by Cheukting** [comment]
@Mariatta Thanks! I have commit your suggestions

File: content/news/cfp-close.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+- [Sayantika](https://www.linkedin.com/in/sayantika-banik/) (English speaking)
+- [Maria Jose Molina-Contreras](https://www.linkedin.com/in/mjmolinacontreras/) (English and Spanish speaking)
+
+We will try to run another round of speaker mentorship when the speakers have been selected and help new speakers get a mentor to help prepare their talks.
+
+Other than selecting and scheduling talks, we will be planning other activities during the conference. Last year, we had panels, discussion sessions, and much more. If you have any great ideas, please feel free to email us at: [pyladiescon@pyladies.com](pyladiescon@pyladies.com)
+
+### We need your support
+
+To make PyLadiesCon a success and to support PyLadies, please consider talking to your employer about [sponsoring PyLaidescon 2024](https://conference.pyladies.com/sponsors/). PyLadiesCon is an annual gathering of all PyLadies globally, this is a great opportunity to give back to the community that is dedicated to supporting the underrepresented groups in the tech community.
```
Comment (Mariatta) [inline content/news/cfp-close.md:56]:
```suggestion
To make PyLadiesCon a success and to support PyLadies, please consider talking to your employer about [sponsoring PyLadiescon 2024](https://conference.pyladies.com/sponsors/). PyLadiesCon is an annual gathering of all PyLadies globally, this is a great opportunity to give back to the community that is dedicated to supporting the underrepresented groups in the tech community.
```

**Review by Mariatta — APPROVED** [review]
Thanks! Please merge anytime, and announce on social media.

**Pull request merged**